bench/Regex.java

```diff
@@ -0,0 +1,22 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+import me.regexp.RE;
+
+public class Regex {
+
+    public static void main(String args[]) {
+      long start = JVM.monotonicTimeMillis();
+      RE r1 = new RE("\\[([^\\[\\]])*\\]");
+      RE r2 = new RE("(\\d{3})(\\d{4})");
+      String number = "5555231119";
+      boolean m1 = false;
+      boolean m2 = false;
+      for (int i = 0; i < 1000; i++) {
+        m1 = r1.match(number);
+        m2 = r2.match(number);
+      }
+      System.out.println("Matches: " + m1 + " " + m2);
+      System.out.println("REGEX: " + (JVM.monotonicTimeMillis() - start));
```

Comment:
Can you split the total time in the time taken to compile and the time taken to match?

Nit: move this line before the other System.out.println so we don't consider the time spent in System.out.println